src/gui/optionsdlg.ui

```diff
@@ -2622,12 +2622,22 @@
                  </property>
                  <item>
                   <property name="text">
-                   <string>Pause them</string>
+                   <string>Pause torrents</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
-                   <string>Remove them</string>
+                   <string>Remove torrents</string>
+                  </property>
+                 </item>
+                 <item>
+                  <property name="text">
+                   <string>Remove torrents + folders</string>
```

Comment:
Sorry for the misunderstanding, i meant to use _folder_ on the 4th entry only. + _files_ was better here because that's what it does: removes the files of the torrent only whereas the last entry deletes the whole folder i.e. everything in it (yeah, not true for torrents without root folder but still).